convention-develocity-gradle-plugin/plugins/gradle-2-through-4/src/main/java/com/myorg/ConventionGradleEnterpriseGradlePlugin.java

```diff
@@ -12,7 +12,7 @@
 import java.util.Map;
 
 /**
- * An example Gradle plugin for enabling and configuring Gradle Enterprise features (Build Scan only) for
+ * An example Gradle plugin for enabling and configuring Develocity features (Build Scan only) for
  * Gradle versions 2.0 through Gradle v4.10.3.
  */
 public class ConventionGradleEnterpriseGradlePlugin implements Plugin<Project> {
```

Comment:
Shouldn't we rename the class?

Reply:
Nice catch. This was renamed in the `build.gradle` but not here.